spec/requests/reviews_spec.rb

```diff
@@ -0,0 +1,127 @@
+require 'rails_helper'
+
+RSpec.describe 'Reviews API' do
+  # Initialize the test data
+  let!(:restaurant) { create(:restaurant) }
+  let!(:reviews) { create_list(:review, 20, restaurant_id: restaurant.id) }
```

Comment:
Again, generate the assets you need only on a context that really need it

Also, a side note, use generate and not create when possible, generate do not save it to DB and so make tests faster - I don't think it is applicable in that case